source/javascripts/all.js.es6

```diff
@@ -1,13 +1,19 @@
+//= require babel/polyfill
+
+//= require ./app/_search
 //= require ./lib/_energize
 //= require ./app/_toc
+//= require ./app/_remote_load
 //= require ./app/_lang
 
 $(function() {
   loadToc($('#toc'), '.toc-link', '.toc-list-h2', 10);
   setupLanguages($('body').data('languages'));
-  $('.content').imagesLoaded( function() {
+  $('.content').imagesLoaded(async() => {
+    await remoteLoad();
     window.recacheHeights();
     window.refreshToc();
+    setupSearch();
```

Comment:
we want to make sure remote docs are loaded _before_ building search index

Reply:
If it's not obvious from the code why this dependent behavior is necessary, maybe it's worth it to add some comments.